units/systemd-firstboot.service

```diff
@@ -14,7 +14,6 @@ DefaultDependencies=no
 Conflicts=shutdown.target
 After=systemd-remount-fs.service
 Before=systemd-sysusers.service sysinit.target shutdown.target
-ConditionPathIsReadWrite=/etc
```

Comment:
still not grokking this. at the moment when systemd-firstboot runs /etc must be writable because we create files in /etc via atomic create/rename. the condition check runs immediately before the service binary is invoked, hence it should be OK to have